docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -18,64 +18,104 @@ install Glymur on your system.
 
 To interact with the Helioviewer API, users first create a "HelioviewerClient"
 instance. The client instance can then be used to make various queries against
-the API using the same parameters one would use when making a web request.
+the API using the same parameters one would use when making a web request. Note that
+HelioviewerClient is not meant to offer full access to the HelioViewer API and 
+as such it do not support all the inputs that the official API allows. But you can pass
+pass in extra keywords to communicate with the API. Make sure that it is not guaranteed that 
+using the keywords which are not documented will work or not.
```

Comment:
The last sentence needs a bit of a tidy up. Maybe something like: `We cannot guarantee that the undocumented keywords will work.`